ui/src/app/page.tsx

```diff
@@ -1,14 +1,71 @@
-'use client'
+'use client';
 import '@fontsource/roboto/300.css';
 import '@fontsource/roboto/400.css';
 import '@fontsource/roboto/500.css';
 import '@fontsource/roboto/700.css';
-import styles from "./page.module.css";
+import { Container, Typography, Button, Box, Card, CardContent } from '@mui/material';
+import Grid from '@mui/material/Grid2';
+import MonetizationOnIcon from '@mui/icons-material/MonetizationOn';
+import LockIcon from '@mui/icons-material/Lock';
+import InsightsIcon from '@mui/icons-material/Insights';
+import Footer from "@/components/footer/Footer";
 
 export default function Home() {
-  return (
-    <div className={styles.page}>
-      PAGE
-    </div>
-  );
+    return (
+        // Visa puslapio struktūra flex column
+        <Box sx={{ display: 'flex', flexDirection: 'column', minHeight: '100vh', backgroundColor: '#121212', color: '#fff' }}>
```

Comment:
Colors should be globlar instead of providing sperate values as such for later implementation of changing page style.